Makefile

```diff
@@ -62,7 +62,7 @@ go.fqp.gomega := github.com/onsi/gomega
 
 all: peer membersrvc checks
 
-checks: unit-test behave linter
+checks: linter unit-test behave
```

Comment:
I think this makes a lot of sense.  Note that the TravisCI config explicitly calls each of these so if you wanted to change the order for that environment, it will need a tweak there too.